installer/Migration/V5_5_0/Migration.php

```diff
@@ -44,6 +44,16 @@ public function up(): void
 
         $this->updateLangStringVersion($this->getVersion());
 
+        $this->getSchemaHelper()->dropForeignKeys('ohrm_i18n_translate', ['langStringId']);
+        $foreignKeyConstraint = new ForeignKeyConstraint(
+            ['lang_string_id'],
+            'ohrm_i18n_lang_string',
+            ['id'],
+            'langStringId',
+            ['onDelete' => 'CASCADE']
+        );
+        $this->getSchemaHelper()->addForeignKey('ohrm_i18n_translate', $foreignKeyConstraint);
```

Comment:
if someone succeeded here with 5.5 migration, then next migration will run from 5.6. Then they won't have this change. So ideally this should go to 5.6 migrations. but in 5.5 migration also we have to fix the failure. (e.g. delete translation for `this_page_is_being_developed` and  `download_latest_release_with_all_features`)

Reply:
ah right if they deleted the strings themselves then the table won't change when they upgrade to 5.6. I'll add this to 5.6 migrations as well

Reply:
added table change to 5.6 and deleted translations in 5.5